Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -472,12 +472,16 @@ export async function unlockKeychain(keychainPath: string, keychainPwd: string):
  * @param uuid
  */
 export async function deleteProvisioningProfile(uuid: string): Promise<void> {
-    let provProfilePath: string = getProvisioningProfilePath(uuid);
-    tl.warning('Deleting provisioning profile: ' + provProfilePath);
-    if (tl.exist(provProfilePath)) {
-        let deleteProfileCommand: ToolRunner = tl.tool(tl.which('rm', true));
-        deleteProfileCommand.arg(['-f', provProfilePath]);
-        await deleteProfileCommand.exec();
+    let provProfiles: string[] = tl.findMatch(getUserProvisioningProfilesPath(), uuid + '*');
```

Comment:
With this being a public method, it might be good to safeguard that `uuid.trim()` is 1+ characters long or is a valid uuid so we don't somehow delete `.*` under mobile profiles.